app/services/api_classification_service.rb

```diff
@@ -11,6 +11,7 @@ def initialize(test_hosts: TEST_API_HOSTS)
   # @return [Symbol] either `:real` or `:test`
 
   def test(url)
+    return Category.new(uri: nil, test_api: false) unless url
```

Comment:
Is there a test for this change?

Reply:
we can not have URL now, when private_endpoint is empty..